samples/append_rows_proto2.js

```diff
@@ -0,0 +1,313 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(
+  projectId = 'my_project',
+  datasetId = 'my_dataset',
+  tableId = 'my_table'
+) {
+  // [START bigquerystorage_append_rows_raw_proto2]
+  const {BigQueryWriteClient} = require('@google-cloud/bigquery-storage').v1;
+  const sample_data_pb2 = require('./sample_data_pb2.js');
+
+  const protos = require('@google-cloud/bigquery-storage').protos.google.cloud
+    .bigquery.storage.v1;
+  const type = require('@google-cloud/bigquery-storage').protos.google.protobuf
+    .FieldDescriptorProto.Type;
+  const mode = require('@google-cloud/bigquery-storage').protos.google.cloud
+    .bigquery.storage.v1.WriteStream.Type;
+
+  async function appendRowsProto2() {
+    /**
+     * If you make updates to the sample_data.proto protocol buffers definition,
+     * run:
+     *   protoc --js_out=import_style=commonjs,binary:. sample_data.proto
+     * from the /samples directory to generate the sample_data_pb2 module.
+     */
+
+    const writeClient = new BigQueryWriteClient();
+
+    /**
+     * TODO(developer): Uncomment the following lines before running the sample.
+     */
+    // projectId = 'my_project';
+    // datasetId = 'my_dataset';
+    // tableId = 'my_table';
+
+    const parent = `projects/${projectId}/datasets/${datasetId}/tables/${tableId}`;
```

Comment:
Thoughts on moving this below the `protoDescriptor` definition? I had to scroll up to check what it was in the `request` definition on line 129.

Reply:
Ah, yes!